content/rs/security/access-control/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: Access control
+linkTitle: Access control
+description: An overview of access control in Redis Enterprise Software.
+weight: 10
+alwaysopen: false
+categories: ["RS"]
+aliases: ["/rs/administering/access-control/user-roles",
+          "/rs/security/passwords-users-roles/"]
+---
+Role-based access control allows you to scale your Redis deployments while minimizing the overhead involved in managing a cluster with many databases, multiple users, and various access control lists. With RBAC, you can create a role once and then deploy it across multiple databases in the cluster.
+
+You can configure roles with standard or custom templates for database permissions that are based on the Redis ACL syntax. Redis Enterprise allows you to restrict database operations by command, command category, and key pattern.
```

Comment:
We can also do specific pub.sub channels (same as OSS).
But the general statement is correct, so maybe I should not be so needy on the smaller details

Reply:
I added a mention of pub/sub here.